testing/bta/7.3_03-Fabric/build.gradle

```diff
@@ -33,12 +44,16 @@ unimined {
             loader(project.fabric_loader_version)
         }
 
+        replaceLibraryVersion("org.apache.commons", "commons-lang3", "3.17.0")
+
         dependencies {
-            clientImplementation("net.betterthanadventure:FabricGameProvider:1.0.0:client")
-            clientImplementation("net.betterthanadventure:FabricGameProvider:1.0.0:server")
-            clientImplementation("com.github.Better-Than-Adventure:legacy-lwjgl3:1.0.6")
+            implementation("com.github.halotroop2288:fabric-game-provider:1.0.0:client")
```

Comment:
And it's *definitely* going under the BTA Maven group.

This notation is only correct if you're using JitPack anyway... My Maven group is com.halotroop

Reply:
Why is there a seperate client and server jar, anyway?

Reply:
Cause it's a dirty implementation that just calls the main method. It was easier just to copy and paste. I'm working on rewriting it to work like the Minecraft one already. But that is way more complicated...

Reply:
also, these dependencies should probably be injected by the provider, either directly or possibly provided by a json somewhere...